docs/powershell.yaml

```diff
@@ -0,0 +1,20 @@
+Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
```

Comment:
Please rename this file to something like `ssh.ps1` this file is not a yaml, and the name powershell is very non descriptive.
Also if you started the file with something like this, that would help a lot
```
# Powershell script to configure SSH on vSphere Windows VMs
```